infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/TrustedSetup.java

```diff
@@ -34,4 +35,9 @@ private void validateG1Point(final Bytes g1Point) {
   private void validateG2Point(final Bytes g2Point) {
     checkArgument(g2Point.size() == BYTES_PER_G2, "Expected G2 point to be %s bytes", BYTES_PER_G2);
   }
+
+  @Override
+  public String toString() {
+    return "TrustedSetup[" + "g1Points=" + g1Points + ", " + "g2Points=" + g2Points + ']';
+  }
```

Comment:
you can remove the toString override too, rolling back to `record`

Reply:
done, thanks for catching this!